scripts/metadata-import.md

```diff
@@ -0,0 +1,40 @@
+# Metadata-Import script
+<!-- TOC -->
+* [Metadata-Import script](#metadata-import-script)
+  * [What it does?](#what-it-does)
+  * [When we use it?](#when-we-use-it)
+  * [How it Works?](#how-it-works)
+  * [Parameters](#parameters)
+  * [Conclusion/QuickGuide:](#conclusionquickguide)
+<!-- TOC -->
+## What it does?
+
+The Metadata-Import script is designed to import metadata after batch editing in DSpace.
+
+## When we use it?
+
+This script is useful when making large-scale changes to items in DSpace, allowing for easy updating of metadata in
+bulk.
+
+## How it Works?
+
+The Metadata-Import script reads a CSV file containing metadata to import, and updates the relevant items in DSpace with
+the new metadata. This can be useful after batch editing a large number of items.
```

Comment:
"For more information, see the [Batch Metadata Editing](https://wiki.lyrasis.org/display/DSDOC7x/Batch+Metadata+Editing) documentation"